model/src/main/java/de/danoeh/antennapod/model/feed/FeedItem.java

```diff
@@ -58,6 +63,11 @@ public class FeedItem extends FeedComponent implements Serializable {
      * */
     private final boolean hasChapters;
 
+    /**
+     * Is true if database or feeditem has podcast:transcript
+     */
+    private boolean hasTranscript;
```

Comment:
This makes me think of the case you presented yesterday @tonytamsf: what if a transcript doesn't exist in `feedItem` now but does in the future. I reckon that's to be updated on feed refresh, like other (meta)data gets updated at that point (as long as it's still considered the same episode, that is).

Reply:
Yes, that case will be handled

- user hits refresh
- we will check for the transcript
- flip the switch `hasTranscript`

Reply:
Why do we need this variable? Can't we just check if the hash table is empty in the getter method?

Reply:
I can take this out for now, having just URL is not enough, we'll need to download and make sure the format is correct.